### DISCUSSION TITLE
Intellibrite not setting to correct colors or themes

### DISCUSSION THREAD

**Discussion by cmcollander** [tagyoureit/nodejs-poolController#901] [Nixie]
Yesterday I finally was able to wire my single Intellibrite to my relay. I removed the switch from the junction box, and passed those lines as a switch-line to the Nixie box. It correctly turns on and off. But any attempt to assign a color or a theme results in the incorrect color or theme. I've been unable to find a pattern or correlation between what I select and what occurs. Has anybody else found this to be a problem?

**Comment by tagyoureit** [comment]
You should be able to count the number of times the light physically turns on and off.  It should match the following:

![2024-02-22_11-20-25](https://github.com/tagyoureit/nodejs-poolController/assets/1646840/c31bffab-09ad-43ed-8af5-79134a0976cb)
